index.js

```diff
@@ -2,24 +2,19 @@
  * Copyright 2015, Yahoo! Inc.
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
-'use strict';
+
+import listenLib from './dist/lib/listen';
+import subscribeLib from './dist/subscribe';
+import unsubscribeLib from './dist/unsubscribe';
```

Comment:
@redonkulus are we able to use `import` here in `index.js` given that it might be used server side?

Reply:
You would need to compile it back to require first, since node.js doesn't support it.  You could move `index.js` under the `src` folder and compile that back to the root. 

Another option is to have two files, `index.js` and `index.es.js` (or some other name). Where `index.js` uses `require` and `index.es.js` uses import. Then in the package.json you can set `main` to `index.js` and `module` to `index.es.js`.